README.md

```diff
@@ -29,21 +29,22 @@ For the documentation see the [WIKI](https://github.com/MurhafSousli/ngx-sharebu
 ## Available buttons
 
 1. Facebook
-2. Twitter
-3. LinkedIn
-4. Pinterest
-5. WhatsApp
-6. Messenger
-7. Reddit
-8. Tumblr
-9. Telegram
-10. Email
-11. SMS
-12. VK
-13. Mix
-14. Xing
-15. Copy link
-16. Print
+1. Twitter
```

Comment:
Can you fix the numbers?

Reply:
I changed it to use the ordered list like this: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#lists